project/src/components/app/app.tsx

```diff
@@ -7,19 +7,25 @@ import FavoritesPage from '../../pages/favorites-page/favorites-page';
 import NotFoundPage from '../../pages/not-found-page/not-found-page';
 import RoomPage from '../../pages/room-page/room-page';
 import PrivateRoute from '../private-route/private-route';
+import { Offers } from '../../types/offer';
+import { Reviews } from '../../types/review';
 
 type AppProps = {
+  offers: Offers;
+  favoriteOffers: Offers;
+  reviwes: Reviews;
   rentalOffersNumber: number;
 }
 
 function App(props: AppProps): JSX.Element {
```

Comment:
Лучше сразу делать деструктуризацию, если ревьюшки не нужны просто можешь не доставать их
```suggestion
function App({ offers, favoriteOffers }: AppProps): JSX.Element {
```